commands.py

```
@@ -0,0 +1,242 @@
+
+def discover_command(ctx, directory: str, max_files: int = 10, max_lines: int = 100, output_format: str = "table") -> None:
+    """Discover available fields in log files.
+
+    Args:
+        ctx: Click context
+        directory: Directory containing log files to analyze
+        max_files: Maximum number of files to sample
+        max_lines: Maximum number of lines to sample per file
+        output_format: Output format (table, list, or json)
+    """
+    import glob
```

Pylint (3.3.7)

[convention] Import outside toplevel (glob)

(C0415)